### ISSUE TITLE
[Find In Files]: Files Search Count for a single character and for a word differs

### ISSUE THREAD

**Issue by rawat11** [adobe/brackets#11471]
Steps:
1) Launch Brackets
2) Open find in files and type any charcater eg m
3) Again do find in files and search for "ma"

Result : The searched files number for m is less than the count for "ma"

**Comment by abose** [comment]
That is expected based on the implementation.
Currently we have a hard limit on 100,000 results across all files and search will stop as soon as over 100k results are got. 
So what is happening in the case is,
1. whem `m` is searched, it found say 108k results in 8 files and returned.
2. a search for ma could then find 120k across 200 files and then return.

**Comment by rroshan1** [comment]
Yes, this is as per expectation. Please note that the phrase `over X results in Y files` would be mentioned in the panel in your case and hence the values of X in this case should not be compared until it is not preceded by "over".
Please close the issue if you are okay with the explanation.

**Comment by rawat11** [comment]
yes, I understand that and over X results in Y files defines it as well but 100,000 results is our limitation to not the display the all results.
Some results are left unidentified. Is there any way to see the results which were not displayed as part of search results

**Comment by abose** [comment]
yes, yu can filter the search by directories to narrow the search, it is anyway not a good thing to find something from a list of 100,000+, so i think the filtration workflow will be much better in that case.
![image](https://cloud.githubusercontent.com/assets/5336369/8899680/2ca62d80-3455-11e5-9c2a-9c47772941b2.png)
